### DISCUSSION TITLE
Progress displayed wrong

### DISCUSSION THREAD

**Discussion by DatMayo** [cssbattle/forum#262] [Bugs reports]
I signed up yesterday via GitHub. Butt all courses are shown as finished :/
Is there a way to reset my progress?
![2022-05-16 10_24_26-Settings _ CSSBattle](https://user-images.githubusercontent.com/1522607/168550680-7f4c8b70-d42d-4cbe-b98c-d93629c1fdde.png)

**Comment by SelenIT** [comment]
It's not individual progress of a player. It's a log of events. Besides the constantly updated global leaderboard, there are specific events ("[Battles](https://cssbattle.dev/blog/introducing-next-battle)") that run for a limited time, new targets arrive during these events, and players get their chance to write their names into the leaderboard of the event, which gets freezed when the event closes and stays unchanged then. Currently there is no such active event (all past Battles are finished), so players can compete for the global leaderboard only, but hopefully a new Battle will start soon.
